compose.go

```diff
@@ -89,6 +105,46 @@ func (dc *LocalDockerCompose) getDockerComposeEnvironment() map[string]string {
 	return environment
 }
 
+func (dc *LocalDockerCompose) applyStrategyToRunningContainer() error {
+
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		return err
+	}
+
+	for k := range dc.WaitStrategyMap {
+		// Docker compose appends "_1" to every started service by default. Trimming to match running docker container
+		f := filters.NewArgs(filters.Arg("name", strings.TrimSuffix(k.service, "_1")),
+			filters.Arg("publish", strconv.Itoa(k.publishedPort)))
```

Comment:
Nit: indent
```suggestion
		filters.Arg("publish", strconv.Itoa(k.publishedPort)))
```

Reply:
This caused static analysis to fail because the indent wasn't compatible with it's rules: https://github.com/shashank-g172/testcontainers-go/actions/runs/355915148 

Reverted the change in a3e4ee63e6295639f072fd8ae49573ded4e99f4e

Reply:
Argg! My visual inspection thought that line was part of another line, not parameters for L117 method. Sorry about that